client/my-sites/plans/jetpack-plans/product-store/featured-item-card/style.scss

```diff
@@ -64,6 +74,19 @@
 	@media (max-width: $break-mobile) {
 		width: auto;
 	}
+
+	.featured-item-card--vertical & {
+		height: 100%;
+		max-width: 330px;
+		min-width: 250px;
+
+		@media only screen and (min-width: 1125px) {
```

Comment:
Are there any breakpoints in the `@wordpress/base-styles/breakpoints` package that we could use here instead of a hardcoded one? There should be one that's at least close enough 🤔